### ISSUE TITLE
[WebBrowser] Cryptic behavior when guided access is enabled

### ISSUE THREAD

**Issue by EvanBacon** [expo/expo#7387]
[`SFAuthenticationSession` doesn't work with guided access](http://www.openradar.appspot.com/40809553) (rdar://40809553). We should be throwing an error when `UIAccessibilityIsGuidedAccessEnabled()` returns `YES`.

**Comment by EvanBacon** [comment]
I was mistaken closing this, both `SFAuthenticationSession` and `ASWebAuthenticationSession` don't work with guided access.
